gradle.properties

```diff
@@ -1,17 +1,16 @@
 org.gradle.caching=true
 group=io.ballerina.lib
 version=0.1.2-SNAPSHOT
```

Comment:
Don't we need to bump the module version if we are planning to do a new release on top of U10?